### PR TITLE
EUI-4095 - Apply fix

### PR DESCRIPTION
**Pull request by connorpgpmcelroy**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4095

### Change description ###
Services have requested timestamp for events be in local time

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: src/shared/components/palette/history/event-log/event-log-details.component.spec.ts [hunk 1]
```diff
@@ -98,7 +98,8 @@ describe('EventLogDetails', () => {
 
     expect(rows.length).toBe(6);
 
-    let resultDate = new DatePipe(null).transform(EVENT.timestamp, 'utc', null) +
+    // Note - event logs changed
+    let resultDate = new DatePipe(null).transform(EVENT.timestamp, 'local', null) +
```
Comment (phillip-whitaker) [inline src/shared/components/palette/history/event-log/event-log-details.component.spec.ts:102]: I don't really understand this comment

**Review by phillip-whitaker — DISMISSED** [review]
LGBTQ+

File: src/shared/components/palette/history/event-log/event-log-details.component.spec.ts (hunk 1 above)
Comment (connorpgpmcelroy) [reply src/shared/components/palette/history/event-log/event-log-details.component.spec.ts:102]: Will update comment

**Review by phillip-whitaker — APPROVED** [review]
LGBTQ+

**Review by leepavlouhmcts — APPROVED** [review]

**Review by phillip-whitaker — DISMISSED** [review]

**Review by leepavlouhmcts — APPROVED** [review]

**Review by udaydenduluri33 — APPROVED** [review]
LGTMc

**Pull request merged**